types.go

```diff
@@ -19,6 +19,22 @@ import (
 	"time"
 )
 
+type AlertLevel string
+
+const (
+	Critical  AlertLevel = "critical"
+	Important AlertLevel = "important"
+	Utility   AlertLevel = "utility"
+	None      AlertLevel = "none"
+)
+
+type Dependency struct {
+	Name       string
+	URL        string
+	AlertLevel AlertLevel
+	CheckFunc  CheckFunc
+}
+
 type healthDependency struct {
```

Comment:
better to add deprecation comment for HardDependency field,
e.g.

```
// HardDependency: this was replaced with AlertLevel.
HardDependency bool `json:"hardDependency"`
```

Reply:
updated